compose/src/main/java/com/maplibre/compose/camera/extensions/MapViewCameraDirection.kt

```diff
@@ -0,0 +1,58 @@
+package com.maplibre.compose.camera.extensions
+
+import com.maplibre.compose.camera.CameraState
+import com.maplibre.compose.camera.MapViewCamera
+
+/**
+ * Get the camera's current direction.
+ *
+ * Direction is null while tracking user location with bearing. In this case, the MapView is always
+ * oriented to the user's current direction of travel.
+ *
+ * @return the current direction.
+ */
+fun MapViewCamera.getDirection(): Double? {
+  return when (this.state) {
+    is CameraState.Centered -> this.state.direction
+    is CameraState.TrackingUserLocation -> this.state.direction
+    is CameraState.TrackingUserLocationWithBearing -> null
+  }
+}
+
+/**
+ * Set the camera's direction (0 to 360 degrees).
+ *
+ * This function will ensure that the direction is within the min and max direction values.
+ *
+ * ```kotlin
+ * val camera = remember { mutableStateOf(MapViewCamera()) }
```

Comment:
Don't we have a `rememberSaveableMapViewCamera` helper or something?

Reply:
We sure do! Good call out. Updated 👍